peer/bind.go

```diff
@@ -124,42 +128,46 @@ func (c *BoundChooser) Introspect() introspection.ChooserStatus {
 // BindPeers returns a binder (suitable as an argument to peer.Bind) that
 // binds a peer list to a static list of peers for the duration of its
 // lifecycle.
-func BindPeers(ids []peer.Identifier) Binder {
+func BindPeers(ids []peer.Identifier) peer.Binder {
 	return func(pl peer.List) transport.Lifecycle {
-		return &peersBinder{
+		return &PeersUpdater{
 			once: intsync.Once(),
 			pl:   pl,
 			ids:  ids,
 		}
 	}
 }
 
-type peersBinder struct {
+// PeersUpdater binds a fixed list of peers to a peer list.
+type PeersUpdater struct {
```

Comment:
It is convenient for config tests to be able to validate the object graph implements these types.